### DISCUSSION TITLE
[BUG]Is there a problem with the PK rule?

### DISCUSSION THREAD

**Discussion by toadzhou** [Chia-Network/chia-blockchain#4999] [General]
Why have I been PK with a big family? Why, I want to change my PK. . . This is unscientific, and I will always fail in PK with him. . .

![PK](https://user-images.githubusercontent.com/1249205/118235033-bd74ba00-b4c6-11eb-989d-abdb75dd9dcc.png)

**Comment by toadzhou** [comment]
I checked this is the address of hpool, and if I play it like this, sooner or later it will become the monopoly of his hpool family. What else do ordinary users play? Where is the fairness?

**Comment by toadzhou** [comment]
Let the big mining pools be the nodes in the future, there is no need for our small user nodes to exist.

**Comment by Plopsi** [comment]
errm,  that's only the info that a block was not farmed by you but another one - in your filtered case by this address, which is hpool. And yes they are annoying, but that's how such decentralised networks work. If everything goes proper next week chia will officially support pools in a prototype implementation. So be strong an patient, and never give away your mnemonics or private keys :)
